cli/pcluster/cli.py

```diff
@@ -38,6 +38,10 @@ def command(args, extra_args):
     pcluster.command(args, extra_args)
 
 
+def connect_to_dcv(args):
```

Comment:
I'd align the name of the function with the name of the cli command, so "dcv".

to be aligned with the other function/commands in the cli module.